frontend/lib/src/components/widgets/DataFrame/DataFrame.tsx

```diff
@@ -145,6 +146,10 @@ function DataFrame({
 
   const { theme, headerIcons, tableBorderRadius } = useCustomTheme()
 
+  const {
+    libConfig: { enforceDownloadInNewTab = false }, // Default to false, if no libConfig, e.g. for tests
```

Comment:
nit: I believe we can also just call that within the `useDataExporter`. Thought, I don't have any strong opinion if its better to get the context here or within the `useDataExporter` hook.

Reply:
Yeah, I just thought that not having nested hooks could be easier to comprehend, so keep it in DataFrame component, and pass to useDataExperter as parameter a little bit more preferable for me, although I also don't have super strong opinion here.